packages/atlas/src/views/studio/VideoWorkspace/VideoForm/VideoForm.tsx

```diff
@@ -462,26 +467,36 @@ export const VideoForm: React.FC<VideoFormProps> = React.memo(({ onSubmit, setFo
                   )
                 }
               />
-              <FormField title="NFT creator royalties">
+              <FormField title="Set creator's royalties">
                 <TextField
                   type="number"
                   {...register('nftRoyaltiesPercent', {
                     valueAsNumber: true,
+                    validate: (value) => {
+                      if (value && value < nftMinCreatorRoyaltyPercentage) {
+                        return `Creator royalties must be at least ${nftMinCreatorRoyaltyPercentage}%`
+                      }
+                    },
                     required: {
                       value: watch('mintNft'),
                       message: 'Creator royalties must be set',
                     },
                     min: {
                       value: 0,
-                      message: 'Creator royalties cannot be lower than 0',
+                      message: 'Creator royalties cannot be lower than 0%',
```

Comment:
As I understand we should use here `nftMinCreatorRoyaltyPercentage` both in the value and in the message

Reply:
Yeah it's a good point, actually the minimum value for royalties is 1%. We allow 0 here, because we interpret this as "royalties disabled". But what if user provides `0.5`? Let me ping @drillprop on Slack, we can discuss

Reply:
Actually `nftMinCreatorRoyaltyPercentage` is `1`, and I handle "0.5 scenaro" on the line `475`. In these lines, I only make sure that no one set a negative value.  
We decided that we will wait for a response from Kuba because this is a kinda UX issue.